src/libs/conduit/python/conduit_python.cpp

```diff
@@ -6041,11 +6246,15 @@ PyConduit_Node_print_detailed(PyConduit_Node *self)
 {
     std::ostringstream oss;
     self->node->to_string_stream(oss,"conduit_json");
+    #ifdef Py_LIMITED_API
+    std::fprintf(stdout,"%s",oss.str().c_str());
```

Comment:
It seems that this can just always be used…the Python juggling seems excessive anyways.